homeassistant/components/legrand_rflc/light.py

```diff
@@ -0,0 +1,180 @@
+"""The Legrand RFLC integration light platform."""
+
+from collections.abc import Mapping
+from typing import Final
+
+from homeassistant import config_entries
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_TRANSITION,
+    COLOR_MODE_BRIGHTNESS,
+    SUPPORT_TRANSITION,
+    LightEntity,
+)
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+
+class _LegrandRflcSwitch(LightEntity):
+    _attr_should_poll = False
+
+    def __init__(self, hub, zid: int, properties: Mapping):
+        self._hub = hub
+        self._attr_name = properties[hub.NAME]
+        self._attr_is_on = properties[hub.POWER]
+        self._zid = zid
+        hub.on(
+            f"{hub.EVENT_ZONE_PROPERTIES_CHANGED}:{zid}",
+            self._zone_properties_changed,
+        )
+        hub.on(hub.EVENT_CONNECTED, self._available)
+        hub.on(hub.EVENT_DISCONNECTED, self._available)
+        hub.on(hub.EVENT_AUTHENTICATED, self._available)
+        hub.on(hub.EVENT_UNAUTHENTICATED, self._available)
+
+    @property
+    def unique_id(self) -> str:
+        return f"{self._hub.host()}:{self._zid}"
```

Comment:
hub is created in __init__:async_setup_entry with the entry.data[CONF_HOST] that was configured/specified by the user.

I do not want to use something truly unique because I want to be able to replace a failed LC7001 hub with another (identically configured one) without having to reconfigure everything in homeassistant. I want to use the host name (or IP address) as the user configured it.